### DISCUSSION TITLE
Email address verification

### DISCUSSION THREAD

**Discussion by phutti** [liedekef/events-made-easy#630] [Q&A] [answered]
Hi all,

it happens often, that email addresses are incorrect typed in registration forms.

Is it possible to force a retyping in an additional form field, verify the identity and  show a hint if they do not match?

Thanks for your help in advance.

Kind regards 
Peter

**Comment by f14m07** [comment]
I had this one time on my local site. It was because I had lost my internet connection

**Comment by phutti** [comment]
Perhaps I didn't describe my problem correct:

I use EME for registration to sports events on our homepage. Interested clients use an EME registration form to register for the event. One field in the registration form is the mail address where often typos happen.
I have seen on several other websites that the clients have to write their mail addresses twice to check for typos.

Hopefully this makes my request a bit more clear. 

Perhaps this is possible using [eme_if... /eme_if]...
Is it possible to compare stings in that way?

**Comment by liedekef** [comment, marked as answer]
EME doesn't foresee that feature. I've seen it on many forms and saw it beaten by copy/paste so many times, so asking 2 times for the same email seems pointless to me.
For now, you could add a custom field and do the comparison in the eme filter eme_eval_booking_form_post_filter

**Reply by phutti** [reply]
Thanks for your hint.
Copy/paste ... yes, would often be done. But perhaps the submitter has then an additional look on his/her input.
Nevertheless I will try it with your proposal.
